### ISSUE TITLE
Cannot use arrow keys to navigate between suggestions

### ISSUE THREAD

**Issue by diegohaz** [btripoloni/chisele#1]
At least in demo, on Chrome 39, I cannot use the arrow keys on suggestions. Also, tab and enter/return keys don't have any behaviour, as it is common in autocomplete features.

**Comment by btripoloni** [comment]
This is true, Chisele is not completed yet and has a slow development, but i'm working in this, please if you are using Chisele on your project let me know to help you, and you can tell me what features you want see on Chisele.

Thanks.
